client/landing/domains/registrant-verification/index.jsx

```diff
@@ -27,8 +27,6 @@ class RegistrantVerificationPage extends Component {
 
 	state = {
 		isLoading: true,
-		success: false,
-		error: false,
```

Comment:
This could also be a direct `state = this.getLoadingState()` assignment. And the constructor can be removed.

Reply:
Good point. Addressed as part of 7c42937dd0219cfcdeb6bed8296c7f183b7e498d